expo/gbe/functions.py

```diff
@@ -67,6 +68,23 @@ def mail_to_group(subject, message, group_name):
     return None
 
 
+def mail_schedule_to_user(subject, user, start_message, end_message):
```

Comment:
me either... I didn't commit this intentionally - as soon as I synced with GBE-665 this showed up.